cpp/benchmarks/profiling.md

```diff
@@ -23,8 +23,8 @@ How to use these tools, the general optimization loop:
 gperftools (formerly google performance tools) is a suite of performance tools. Here the focus is on the [profiler](https://gperftools.github.io/gperftools/cpuprofile.html). It's a small sampling profiler that is quick to setup and use on most systems. It's integrated into our build system for even greater convenience.
 
 Basic steps:
-1. Install gperftools. It's available in many package managers (apt packages `google-perftools` and `libgoogle-perftools-dev`, spack or homebrew package `gperftools`). E.g., on the DLR-SC hpda cluster (insert the compiler version you are using): `module load spack-user; spack install gperftools%gcc@13.1.0; module load gperftools`.
-2. configure memilio with cmake variable `MEMILIO_ENABLE_PROFILING=ON`.
+1. Install gperftools. It's available in many package managers (apt packages `google-perftools` and `libgoogle-perftools-dev`, spack or homebrew package `gperftools`). E.g., on the DLR-SC hpda cluster (insert the compiler version you are using): `module load spack-user; module load PrgEnv/gcc13-openmpi; spack install gperftools%gcc@13.3.0; module load gperftools`.
+2. configure memilio with cmake variable `MEMILIO_ENABLE_PROFILING=ON`. Note that compiling with profiling enabled does not incur any runtime overhead unless profling is also enabled at runtime (see step 4), so developers can just enable it always for convenience.
```

Comment:
> Note that compiling with profiling enabled does not incur any runtime overhead 

I wonder, is that also true for RelWithDebugInfo? Do they work the same?
(I am just curious, no changes needed)

Reply:
I believe it is, the binary output of `RelWithDebInfo` should be the same as `Release` with some (maybe unused) source code positions.